arch/arm/boot/dts/overlays/README

```diff
@@ -1107,6 +1107,13 @@ Load:   dtoverlay=raspidac3
 Params: <None>
 
 
+Name:   rotary-encoder
+Info:   Overlay for GPIO connected rotary encoder.
+Load:   dtoverlay=rotary-encoder,<param>=<val>
+Params: rotary0_pin_a           GPIO connected to rotary encoder channel A (default 4).
```

Comment:
This line and the next are too long (there is an 80-character limit). Push "(default 4)." onto the next line, indented under GPIO.